### PR TITLE
DL 14981: Remove Unused Feature Switch

### PR DESCRIPTION
**Pull request by diksha-diksha**

### CODE REVIEW

File: app/config/featureToggles/FeatureSwitch.scala [hunk 1]
```diff
@@ -23,9 +23,7 @@ object FeatureSwitch {
   val featureSwitches: Seq[FeatureSwitch] = Seq(
     AccessMTD,
     IvUpliftSwitch,
-    PptSwitch,
```
Comment (chaithupejjai) [inline app/config/featureToggles/FeatureSwitch.scala:26]: Seems to be all checks passed. so I do not think ATs would have impacted. Please check All ATS are okay or not?

File: app/controllers/OtherTaxesController.scala [hunk 2]
```diff
@@ -17,7 +17,7 @@
 package controllers
 
 import config.FrontendAppConfig
-import config.featureToggles.FeatureSwitch.{ECLSwitch, NewADPJourney, Pillar2Switch, PptSwitch}
+import config.featureToggles.FeatureSwitch.{ECLSwitch, Pillar2Switch}
```
Comment (chaithupejjai) [inline app/controllers/OtherTaxesController.scala:20]: I strongly request you to have a short call with @shankaran-hmrc  tomorrow reagrding this befor merge. He spent good amount of time for Switches, he can help you if any repos need this change with respect to this. I just skimmed through the code. All looks good to me.

**Review by shankaran-hmrc — APPROVED** [review]
reviewed PR look good and verified UT AT and requested to raise config PRs

File: app/config/featureToggles/FeatureSwitch.scala (hunk 1 above)
Comment (diksha-diksha) [reply app/config/featureToggles/FeatureSwitch.scala:26]: All ATs are checked and resulted in no failure.

**Pull request merged**